<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>two_player</class>
 <widget class="QMainWindow" name="two_player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="white_number">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="black_number">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>370</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Color</string>
    </property>
    <widget class="QRadioButton" name="radioButton_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Black</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../build-Login-Desktop_Qt_5_1_0_MinGW_32bit-Debug/Black_Circle.png</normaloff>../build-Login-Desktop_Qt_5_1_0_MinGW_32bit-Debug/Black_Circle.png</iconset>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>White</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../build-Login-Desktop_Qt_5_1_0_MinGW_32bit-Debug/white-circle.png</normaloff>../build-Login-Desktop_Qt_5_1_0_MinGW_32bit-Debug/white-circle.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="white_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-Login-Desktop_Qt_5_1_0_MinGW_32bit-Debug/download.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="black_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-Login-Desktop_Qt_5_1_0_MinGW_32bit-Debug/Black_Circle.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>71</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPushButton" name="Start_3">
       <property name="text">
        <string>Start Game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Save_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save Game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Load_3">
       <property name="text">
        <string>Load Game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="End_3">
       <property name="text">
        <string>End Game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Back_3">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>139</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLineEdit" name="x_3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="y_3"/>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
